modules/tinymce/src/plugins/table/test/ts/browser/ApplyCellStyleCommandTest.ts

```diff
@@ -0,0 +1,148 @@
+import { Step, Log, ApproxStructure, Pipeline } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import { Obj } from '@ephox/katamari';
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import SilverTheme from 'tinymce/themes/silver/Theme';
+import * as TableTestUtils from '../module/test/TableTestUtils';
+
+UnitTest.asynctest('browser.tinymce.plugins.table.ApplyCellStyleCommandTest', (success, failure) => {
+  Plugin();
+  SilverTheme();
+
+  TinyLoader.setup((editor: Editor, onSuccess, onFailure) => {
+    const tinyApis = TinyApis(editor);
+
+    const beforeTable = ({ cell1 = false, cell2 = false } = {}) =>
+      '<table style="border-collapse: collapse; width: 100%;" border="1">' +
+      '<tbody>' +
+      '<tr>' +
+      `<td style="width: 50%;" ${cell1 ? 'data-mce-selected="1"' : ''}>a</td>` +
+      `<td style="width: 50%;" ${cell2 ? 'data-mce-selected="1"' : ''}>b</td>` +
+      '</tr>' +
+      '</tbody>' +
+      '</table>';
+
+    const mapStyles = (styles: Record<string, string>, str) => Obj.map(styles, (val, _key) => str.is(val));
+
+    const sAsserTableCellStructure = (styles: Record<string, string> = {}, { cell1 = false, cell2 = false, cell3 = false, cell4 = false } = {}) =>
```

Comment:
This destructing while setting defaults in a function argument is very hard to read, so this probably shouldn't be done. Please just make this a single argument and then destructure it on a new line. 

Also there's a typo in the function name (missing `t` in `Assert`).

Reply:
Also, you don't look to even be using `cell3` and `cell4` below and then I never see you pass in `cell2` as an argument here, so this looks like it needs some cleanup.

I'm also not convinced this makes sense to be an object, it'd likely be cleaner and more readable to just use arguments directly.

Reply:
I have cleaned up both of the test files so hopefully they are good now.